pkgs/by-name/db/dbtpl/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  buildPackages,
+  versionCheckHook,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "dbtpl";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "xo";
+    repo = "dbtpl";
+    tag = "v${version}";
+    hash = "sha256-r0QIgfDSt7HWnIDnJWGbwkqkXWYWGXoF5H/+zS6gEtE=";
+  };
+
+  vendorHash = "sha256-5ISkkGnuhmmFVGiHrgPD4LepFNjHUab28Siuqt6ZKdA=";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  preBuild = ''
```

```suggestion
  postPatch = ''
```
A bit more idiomatic to edit source in a patch-adjacent phase